nengo/utils/matplotlib.py

```diff
@@ -200,23 +198,3 @@ def rasterplot(time, spikes, ax=None, use_eventplot=False, **kwargs):  # noqa
     ax.yaxis.set_ticks_position("none")
 
     return ax
-
-
-def plot_tuning_curves(ensemble, sim, connection=None, ax=None):
```

Comment:
Are we okay to just get rid of this, or do we want to deprecate and remove in 4.0 just in case somebody is using it?

Reply:
Up to you, I suggested removing it because I didn't realize it existed and we don't use it anywhere in our examples (though we do plot tuning curves, just not with this function).